tests/CacheCompat/CommonCache.Test.Common/CommandLineOptions.cs

```diff
@@ -25,10 +25,20 @@
 // 
 // ------------------------------------------------------------------------------
 
+using System;
 using CommandLine;
 
 namespace CommonCache.Test.Common
 {
+    [Flags]
+    public enum CacheStorageType
+    {
+        None = 0,
+        Adal = 1,
```

Comment:
Some places you have Adal and then AdalV3. I would either go w/just Adal or AdalV_ like you did in Msal.